shardingsphere-sql-parser/shardingsphere-sql-parser-engine/src/main/java/org/apache/shardingsphere/sql/parser/core/visitor/SQLVisitorFactory.java

```diff
@@ -62,7 +62,7 @@
             case RL:
                 return (ParseTreeVisitor) visitorFacade.getRLVisitorClass().getConstructor().newInstance();
             default:
-                throw new SQLParsingException("Can not support SQL statement type: `%s`", type);
+                throw new SQLParsingException(String.format("Can not support SQL statement type: `%s`", type));
```

Comment:
The SQLParsingException has already use String.format inside, it is unnecessary format string outside constructor